components/places/src/types.rs

```diff
@@ -169,6 +169,43 @@ impl<'de> serde::Deserialize<'de> for VisitTransition {
     }
 }
 
+// Re SyncStatus - note that:
+// * logins_sql has synced=0, changed=1, new=2
+// * desktop bookmarks has unknown=0, new=1, normal=2
+// This is "places", so eventually bookmarks will have a status - should history and bookmarks share it?
```

Comment:
That's an interesting idea! Sharing a `syncStatus` for history and bookmarks could be tricky (if you're syncing one but not the other, if you restore bookmarks but not history), but I haven't thought much about it.

Reply:
> That's an interesting idea! Sharing a `syncStatus` for history and bookmarks could be tricky

FWIW, I just meant the SyncStatus enum could be shared, not that we'd try and share the fields somehow.